pkg/alertmanager/statefulset.go

```diff
@@ -488,13 +502,17 @@ func makeStatefulSetSpec(a *monitoringv1.Alertmanager, config Config, tlsAssetSe
 		},
 	}
 
-	reloadWatchDirs := []string{alertmanagerConfigDir}
+	reloadWatchDirs := []string{}
```

Comment:
I'm not sure why we had `reloadWatchDirs` and `watchedDirectories` while they are identical copies... Let's get rid of `reloadWatchDirs`.
 
```suggestion
	var watchedDirectories []string
```